lib/DropdownMenu/styles.scss

```diff
@@ -3,7 +3,8 @@
    ========================================================================== */
 $dropdown-menu-caret-margin: $layout-spacing-base/2 !default;
 $dropdown-menu-icon-size: $typo-size-small !default;
-$dropdown-menu-link-padding: 7px 15px !default;
+$dropdown-menu-link-padding: $layout-spacing-base/4 $layout-spacing-base/1.5 !default;
+$dropdown-menu-font-size: 13px !default;
```

Comment:
Should this use a setting rather than a hardcoded value of 13px?

Reply:
Only problem is $typo-size-base feels too big and $typo-size-small is too small lol. I mean I could always make a setting in between them two what you think @kyleharper ?

Reply:
Get some design direction but we shouldn't be using hard coded values. We've had many issues in the past where hardcoded values have made the codebase less predictable and harder to work with.

Either;

- add a new setting using language and direction from designers
- derive font-size using a setting (we do this with spacing a lot)